service_capacity_modeling/capacity_planner.py

```diff
@@ -377,19 +377,30 @@ def _plan_certain(
             if per_instance_mem > instance.ram_gib:
                 continue
 
-            for drive in hardware.drives.values():
-                if not _allow_hardware(drive.name, drive.lifecycle, drives, lifecycles):
-                    continue
-
+            if "evcache" in model_name:
```

Comment:
This is the generic planner interface, it's the wrong level of abstraction to add model names. The model itself will be shown all possible instance+drive combinations, it can ignore ones it doesn't want.